src/components/Header.js

```diff
@@ -6,6 +6,13 @@ const Header = () => {
     lineHeight: '1.5em',
   };
 
+  /*
+  const headerStyle2 = {
+    padding: '0',
+    lineHeight: '1.5em',
+  };
+  */
+
```

Comment:
- [ ] Avoid leaving commented-out code in the source code as it is bad practice, it takes up space, causes confusion, and leads to maintenance issues if it's not removed